Monika After Story/game/script-topics.rpy

```diff
@@ -8922,74 +8922,92 @@ init 5 python:
-        m 2efo "But people who aren't conventionally attractive end up in relationships all the time!"
-        m 2tfc "So the idea of some kind of universal beauty standard where, if you fall short, you're doomed to be forever alone..."
-        m 2efw "It just doesn't make any sense to me!"
-    else:
-        m "At first I figured that it must be her metabolism keeping her from putting on weight."
-        m 1eud "But when I learned that Sayori wasn't real, I came up with another theory."
-        m 3esd "As a character in a game, she needed to have some sort of visual representation so that you, [player], could see her."
-        m 3eud "So a designer, possibly more than one, must have decided to draw her that way."
-        m 3eub "And since all the images have the same body type, Sayori would always have the same figure no matter what she did."
-        m 1esa "It would have to work like that for Yuri and Natsuki too."
-        m 1esc "They decided that Natsuki should have a petite figure and that Yuri should be taller and...{w=1}more developed."
-        m "But haven't you noticed that the three of them still look similar in a lot of ways?"
-        m 3eud "Like, none of them are muscular or fat..."
-        m 3tkd "...none of them have any kind of physical disability..."
-        m 3tkc "...none of them are bald or have hair shorter than chin length..."
-        m "...and apart from Yuri having cuts on her arms, none of them have anything wrong with their skin."
-        m 2dsc "The designers must have thought that people would find all that stuff really repulsive."
-        m 2efo "But people who aren't conventionally attractive end up in relationships all the time!"
-        m 2lfc "So did they really have to make the other girls look so perfect?"
-        m 2lfd "I mean, it's bad enough that I didn't get a route in the game..."
-        m 2lfw "...but did they really have to make it even harder for me to get chosen?"
-    m 2dfc "..."
-    m 2dkc "..."
+    m 3euc "At first I figured that it must be her metabolism keeping her from putting on weight."
+    m 3esc "But when I learned that our world isn't real, I came up with another theory."
+    m 2euc "As characters in a game, we needed to have some sort of visual representation so that you, the player, could see us."
+    m "Sayori, Natsuki, Yuri, and even me...{w=0.5}{nw}"
+    extend 2rkc "we don't really have bodies like people in your world do. We only have a set of images to represent our appearances and current emotion."
+    m 2eud "So a designer, possibly more than one, must have decided to draw us the way we are."
+    m 2eub "And since all those images have the same body type, Sayori would always have the same figure no matter what she did."
+    m 2dkc "It's...weird to think about. I can feel myself being present in this very room right now, I can look at myself like I presume you would be able to look at yourself, almost as if I had an actual body..."
```

Comment:
That's my personnal feeling as always, but I feel this line and the next are quite immersion breaking, reminding the player she's objectively not anything more than a set of sprites. A way I'd personnaly feel more comfortable with this would be something more akin to `monika_perspective`, where she points out more she lives with the perspective of a real human first and foremost, but can also see the game as we see it.


More specificaly, the 'I presume', the 'almost as if I had an actual body' in the first line and all of the second line clashes with my perception (which I don't think is uncommon even if I ultimately don't know) of who she is and how she lives.